coveo_stew/ci/config.py

```diff
@@ -98,31 +99,36 @@ def get_runner(self, runner_name: str) -> Optional[ContinuousIntegrationRunner]:
         return self._runners.get(runner_name)
 
     def _generate_ci_plans(
-        self, checks: Optional[List[str]], parallel: bool = True
+        self, checks: Optional[List[str]], skips: Optional[List[str]], parallel: bool = True
     ) -> Generator[CIPlan, None, None]:
         """Generates one test plan per environment."""
         checks = [check.lower() for check in checks] if checks else []
+        skips = [skip.lower() for skip in skips] if skips else []
+
+        emojis = cycle(("see_no_evil", "hear_no_evil", "speak_no_evil"))
 
         for environment in self._pyproject.virtual_environments(create_default_if_missing=True):
             runners = []
             for runner in self.runners:
-                if checks and runner.name.lower() not in checks:
+                if checks and runner.name.lower() not in checks or runner.name.lower() in skips:
```

Comment:
```suggestion
                if (checks and runner.name.lower() not in checks) or runner.name.lower() in skips:
```

It's not required because `and` gets resolved before `or` but I find it helpful. This way there's no ambiguity. I've had to look up python's operator precedence list. I'm probably not the only one who's going to be uncertain of the order.